docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -275,13 +275,9 @@ This `Dialog` object has a [messageChild](/javascript/api/office/office.dialog#o
 Consider a scenario in which the UI of the dialog is related to the currently active worksheet and that worksheet's position relative to the other worksheets. In the following example, `sheetPropertiesChanged` sends Excel worksheet properties to the dialog box. In this case, the current worksheet is named "My Sheet" and it's the second sheet in the workbook. The data is encapsulated in an object and stringified so that it can be passed to `messageChild`.
 
 ```javascript
-function sheetPropertiesChanged() {
-    const messageToDialog = JSON.stringify({
-                               name: "My Sheet",
-                               position: 2
-                           });
-
-    dialog.messageChild(messageToDialog);
+function sheetPropertiesChanged() {
+    const messageToDialog = JSON.stringify({ name: "My Sheet", position: 2 });
```

Comment:
Same comment as the reference PR (consider using actual parameters instead of hardcoded data), but the preceding context helps some.

Reply:
Thanks for the feedback. I replaced the hardcoded data with a method parameter and added some context for getting the active worksheet.